google/cloud/spanner_dbapi/connection.py

```diff
@@ -63,28 +69,64 @@ def __handle_update_ddl(self, ddl_statements):
         """
         Run the list of Data Definition Language (DDL) statements on the underlying
         database. Each DDL statement MUST NOT contain a semicolon.
-        Args:
-            ddl_statements: a list of DDL statements, each without a semicolon.
-        Returns:
-            google.api_core.operation.Operation.result()
+
+        :type ddl_statements: list
+        :param ddl_statements: A list of DDL statements, each without a
+                               semicolon.
         """
         self._raise_if_closed()
         # Synchronously wait on the operation's completion.
         return self.database.update_ddl(ddl_statements).result()
 
     def read_snapshot(self):
+        """Return a Snapshot of the linked Database.
+
+        :rtype: :class:`~google.cloud.spanner_v1.snapshot.Snapshot`
+        :returns: A snapshot of the linked Database.
+        """
         self._raise_if_closed()
         return self.database.snapshot()
 
     def in_transaction(self, fn, *args, **kwargs):
+        """Perform a unit of work in a linked Transaction, retrying on abort.
+
+        :type fn: callable
+        :param fn: takes a required positional argument, the transaction,
+                   and additional positional / keyword arguments as supplied
+                   by the caller.
+
+        :type *args: tuple
+        :param *args: additional positional arguments to be passed to ``fn``.
+
+        :type **kwargs: dict
+        :param **kwargs: (Optional) keyword arguments to be passed to ``fn``.
+                         If passed, "timeout_secs" will be removed and used to
+                         override the default retry timeout which defines
+                         maximum timestamp to continue retrying the transaction.
+
+        :rtype: Any
+        :returns: Runs the given function as it would be a transaction.
+        """
         self._raise_if_closed()
         return self.database.run_in_transaction(fn, *args, **kwargs)
 
     def append_ddl_statement(self, ddl_statement):
+        """Append DDL statement to the existing list of DDL statements in
+        linked database.
+
+        :type ddl_statements: list
+        :param ddl_statements: A list of DDL statements, each without a
+                               semicolon.
+        """
         self._raise_if_closed()
         self._ddl_statements.append(ddl_statement)
 
     def run_prior_DDL_statements(self):
+        """Run prior Operation.
+
+        :rtype: str
+        :returns: Updated statements.
```

Comment:
Updated statements? It doesn't return anything, as I've shown you at a [real example](https://github.com/q-logic/python-spanner-django/pull/13#discussion_r500174547).